src/olympia/users/tests/test_models.py

```diff
@@ -1111,7 +1126,7 @@ def test_create_superuser(self):
         )
         assert user.pk is not None
         Group.objects.get(name="Admins") in user.groups.all()
-        assert user.is_staff
+        assert not user.is_staff  # Not a mozilla.com email...
```

Comment:
so... if you don't have a @mozilla.com email address you can't be a superuser, even locally?  Doesn't that lock out any contribution that involves django admin?

Reply:
If you don't have a mozilla.com email you can be a superuser, but you won't have `is_staff` so you won't have access to the admin. With https://github.com/mozilla/addons-server/issues/15701 you can easily create a mozilla.com user to gain access the admin locally, so I don't think it's a big problem.